Xamarin.PropertyEditing.Mac/Controls/CharEditorControl.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Xamarin.PropertyEditing.Mac.Resources;
+using Xamarin.PropertyEditing.ViewModels;
+
+namespace Xamarin.PropertyEditing.Mac
+{
+	internal class CharEditorControl
+		: EntryPropertyEditor<char>
+	{
+		public CharEditorControl (IHostResourceProvider hostResources)
+			: base (hostResources)
+		{
+		}
+
+		protected override EntryPropertyEditorDelegate<char> CreateDelegate (PropertyViewModel<char> viewModel)
+		{
+			return new CharDelegate (viewModel);
+		}
+
+		protected override void UpdateAccessibilityValues ()
+		{
+			base.UpdateAccessibilityValues ();
+			Entry.AccessibilityTitle = string.Format (LocalizationResources.AccessibilityTimeSpan, ViewModel.Property.Name);
```

Comment:
I suspect this might be a copy/paste error, but the `Entry.AccessibilityTitle` should probably be assigned `LocalizationResources.AccessibilityChar`?